third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -0,0 +1,53 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccLoggingMetricUpdate(t *testing.T) {
```

Comment:
```suggestion
func TestAccLoggingMetric_update(t *testing.T) {
```

I think the resource you worked on before had some different naming than normal, most tests will be named `TestAcc{{Product}}{{Resource}}_{{Description}}` where the description is `camelCased`.